documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -115,7 +115,7 @@ tls:
   trustedCertificates: []
 ----
 
-Similarly, you can use the `tlstrustedCertificates` property in the configuration for `oauth`, `keycloak`, and `opa` authentication and authorization types that integrate with authorization servers.
+Similarly, you can use the `tlstrustedCertificates` property in the configuration for `oauth`, `keycloak`, and `custom` authentication and authorization types that integrate with authorization servers.
```

Comment:
I guess I'm fine if you remove OPA from here. But neither `type? custom` authentication not authorization supports `tlstrustedCertificates`. They would use additional volumes to mount anything like TLS truststore. So you should not replace the `opa` type with the `custom` type but remove it (or keep the `opa` type).